frontend/src/components/NaiveDatePicker.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <!-- use this component for mutation of display dates (i.e. not TZ aware) -->
```

Comment:
@FrenchGithubUser we discussed this briefly in Discord but I wanted to go a different route - JS does not have a concept of a naive date time object and that split between how it'll be handled in the backend vs frontend will be a source of frustrating bugs. 

For now it feels like a safer approach to just keep raw date strings around.

I think the way I wrote this component is almost definitely not idiomatic Vue - I would love guidance on how to make it better